src/Components/Publishing/Article.tsx

```diff
@@ -185,31 +194,43 @@ export class Article extends React.Component<ArticleProps, ArticleState> {
           </StandardLayoutParent>
 
           {relatedArticlesForCanvas &&
-            <RelatedArticlesCanvas
-              articles={relatedArticlesForCanvas}
-              isMobile={isMobile}
-              vertical={article.vertical}
-            />}
+            <div>
+              <LineBreak />
+              <RelatedArticlesCanvas
+                articles={relatedArticlesForCanvas}
+                isMobile={isMobile}
+                vertical={article.vertical}
+              />
+            </div>}
 
         </ReadMoreWrapper>
 
         {this.state.isTruncated &&
           <ReadMore
             onClick={this.removeTruncation}
           />}
+
         {this.props.display && (
           <div>
-            <DisplayCanvasBreak />
-            <DisplayCanvas unit={this.props.display.canvas} campaign={campaign} />
+            <LineBreak />
+            {displayOverflows
+              ? <FooterContainerOverflow>
+                <DisplayCanvas unit={this.props.display.canvas} campaign={campaign} />
+              </FooterContainerOverflow>
```

Comment:
Prettier + JSX is so broken

Reply:
Isn't this more that the code itself is pretty verbose for using the tertiary operator?  This could be more readable by choosing a container earlier in the function and using that instead of duplicating the code inside the container, e.g.

```js
const displayOverflows = this.props.display && this.props.display.canvas.layout === "slideshow"
const FooterWrapper = displaysOverflow ? FooterContainerOverflow : FooterContainer

[...]
<FooterWrapper>
   <DisplayCanvas unit={this.props.display.canvas} campaign={campaign} />
</FooterWrapper>
```

I can't really think of an optimal way to format code like this?

Reply:
The code could indeed be structured in many different ways, but this is more of a comment on Prettier breaking in a JSX context due to confusion about indention.

Reply:
That said, I like this pattern (`const FooterWrapper = displaysOverflow ? FooterContainerOverflow : FooterContainer`) a lot. I feel like a lot of devs aren't aware that components can be reassigned like this for readability.

Reply:
Yes into that!  Will keep in mind for the future.